src/assets/src/components/queue.tsx

```diff
@@ -102,56 +102,77 @@ function QueueAttendingNotJoined(props: QueueAttendingProps) {
     );
 }
 
+const MeetingReadyAlert = () =>
+    <Alert variant="success" role="alert">
+        <strong>Your meeting is ready!</strong> Please view the instructions below to join the meeting!
+    </Alert>
+
 const TurnNowAlert = () =>
-    <div className="alert alert-success" role="alert">
-        <strong>It's your turn!</strong> If you haven't already joined the meeting, follow the directions to join it now!
-    </div>
+    <Alert variant="warning" role="alert">
+        <strong>You're next in line,</strong> but your meeting is still being started. When the host is ready, you'll be able to click Join Meeting below.
+    </Alert>
 
 const TurnSoonAlert = () =>
-    <div className="alert alert-warning" role="alert">
-        <strong>Your turn is coming up!</strong> Follow the directions to join the meeting now so you are ready when it's your turn.
-    </div>
+    <Alert variant="warning" role="alert">
+        <strong>Your turn is coming up!</strong> The line may move quickly, so don't go far!
+    </Alert>
```

Comment:
`role` may not be needed here: see https://github.com/react-bootstrap/react-bootstrap/blob/af43b498c03dc45d3e79c3d2c87d6c82b4e2f7d2/src/Alert.tsx#L118-L145

Reply:
Nice find!